services/api/src/models/Users.ts

```diff
@@ -57,6 +60,27 @@ export function getUserQuests(db: Database, id: string): Bluebird<UserQuestsType
         lastPlayed: new Date(result.get('lastPlayed')),
       };
     });
-    return userQuests;
+
+    const metas: Bluebird<void>[] = [];
+    for (const k of Object.keys(userQuests)) {
+      metas.push(db.quests.findOne({
```

Comment:
This does a separate DB lookup for each quest the user has played? Can we do a join here so that this isn't an n^2 lookup?